.github/workflows/build-docs.yaml

```diff
@@ -1,56 +1,107 @@
-name: Build and Deploy
-on:   
-  schedule:
-    - cron: '0 1 * * *'
```

Comment:
Not sure why this runs on a schedule. We can leave it on this cron job, or we build whenever we push to `main`. I am a fan of "building when pushing to `main`", but there might be a reason we do it currently on a scheduled basis.

Reply:
+1 to changes this to trigger on pushes to main.